flags.go

```diff
@@ -134,9 +134,9 @@ func newApp() (app *cli.App) {
 			/////////////////////////
 
 			cli.StringFlag{
-				Name:  "endpoint",
```

Comment:
can we rename the flag to non-prod-endpoint or custom-endpoint and mention that auth will be skipped for custom endpoints?

Reply:
Yeah, that would be good. Thanks!